### PR TITLE
feat(interface): add missing autoupdate

### PR DESCRIPTION
**Pull request by hazcod**

### CODE REVIEW

**Comment by thc202** [comment]
Should that be added to the interface as well?

**Comment by hazcod** [comment]
@thc202 I added it to the interface in this PR?

**Comment by hazcod** [comment]
Good catch @thc202 , added.

**Comment by hazcod** [comment]
Oof. Not sure why DCO is still failing?

**Comment by thc202** [comment]
The report says commit 33637c4 is not signed-off.

File: zap/interface.go [hunk 1]
```diff
@@ -39,6 +40,11 @@ type Interface interface {
 	Users() *Users
 }
 
+// AutoUpdate returns an AutoUpdate client
+func (c *Client) AutoUpdate() *AutoUpdate {
```
Comment (thc202) [inline zap/interface.go]: Could you move this to be in alphabetic order?

**Comment by hazcod** [comment]
@thc202 any idea what i'm doing wrong? After a `git pull`, I do the following:
```
 1008  git commit --amend --signoff --all
 1009  git push --force-with-lease origin patch-1
```

File: zap/interface.go [hunk 2]
```diff
@@ -25,6 +25,7 @@ type Interface interface {
 	Ascan() *Ascan
 	Authentication() *Authentication
 	Authorization() *Authorization
+	AutoUpdate() *AutoUpdate
```
Comment (thc202) [inline zap/interface.go]: Should be `Autoupdate`.

**Comment by thc202** [comment]
I'd suggest:
```
git reset upstream/master
git commit -s
git push origin patch-1 -f
```
to remove merges.

File: zap/interface.go (hunk 1 above)
Comment (thc202) [reply zap/interface.go]: Unresolved this one, it was not reordered.

**Comment by thc202** [comment]
Tweaked.

**Comment by thc202** [comment]
Correct case, missed in the previous push.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**